sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -876,7 +876,7 @@ case class UncacheTable(
 /**
  * The logical plan of the ALTER TABLE ... SET LOCATION command.
  */
-case class AlterTableSetLocation(
+case class SetTableLocation(
```

Comment:
The command incapsulates 2 commands actually: set table location and set partition location. Should we split it to `SetTableLocation` and `SetPartitionLocation` (separately in another PR).

Reply:
SGTM